pkg/sql/conn_executor.go

```diff
@@ -3006,7 +3006,9 @@ func (ex *connExecutor) execCopyIn(
 	}()
 
 	// When we're done, unblock the network connection.
-	defer cmd.CopyDone.Done()
+	defer func() {
```

Comment:
one more idea instead of a buffered channel: could we change this to something like:
```
once.Do(func() { close(ch) })
```
or
```
once.Do(func() { cmd.CopyDone.Done() })
```